packages/react-query-devtools/src/Explorer.tsx

```diff
@@ -124,6 +155,9 @@ export const DefaultRenderer: Renderer = ({
               {subEntries.length} {subEntries.length > 1 ? `items` : `item`}
             </Info>
           </ExpandButton>
+          {copiable ? (<CopyButton>
+            <Copier onClick={() => navigator.clipboard.writeText(displayValue(value))} />
+          </CopyButton>) : null}
```

Comment:
why isn't the click handler on the `button` but on the `span` inside it?

please also give the button:

- a proper `aria-label`
- a `tooltip`

for a11y. Like we also do for the mock offline behaviour button:

https://github.com/TanStack/query/blob/1a1012bf901b99f703b4b219675767ec2bcb8049/packages/react-query-devtools/src/devtools.tsx#L647-L656